components/brave_rewards/resources/ui/components/checkout/formSection/style.ts

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const Box = styled.div`
+  border-radius: 8px;
+  background: #F7F9FB;
+  box-shadow: 1px -1px 6px 0 rgba(12, 23, 33, 0.20);
```

Comment:
I think this should be `0 0 6px 0 rgba(12, 23, 33, 0.20);`. Even though that sketch has this values in the browser it doesn't look the same.